### PR TITLE
Feature: LODs Setup on Dense Meshes [MTT-4451]

### PR DESCRIPTION
**Pull request by jilfranco-unity**
### Description
Initial setup of LODs for some of our more geometrically dense art assets, as well as a lighting re-bake of the boss room level to account for changed geometry. Also went through and cleaned up some geometry that I noticed was a little wonky on the breakable pot, the big torch, and the columns. The new LODs all live in the same DungeonPieces FBX.

Assets given LODs:
* Columns
* Boss Columns
* Heavy Pot
* Breakable Pot
* Banner
* Non animated door frames (the main entrance one and the imp spawner ones)
* Barrier (the little wooden log door for the door frame)
* Roots

### Issue Number(s)

[Jira ticket here](https://jira.unity3d.com/browse/MTT-4451)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

**Comment by SamuelBellomo** [comment]
The only diff I see is between low and high, medium and ultra don't seem to make any difference.
Also, is there a way to auto detect the required quality setting and automatically adjust? (wondering if there's a trivial way to do this, if too complex, disregard)

**Review by SamuelBellomo — DISMISSED** [review]

File: Assets/Models/DungeonPieces.FBX.meta [hunk 1]
```diff
@@ -1,7 +1,7 @@
 fileFormatVersion: 2
```
Comment (fernando-cortez) [inline Assets/Models/DungeonPieces.FBX.meta:1]: When doing a fresh import, I see the warning:
File 'DungeonPieces' has inconsistent LOD naming (_LOD1 found but no _LOD0).

**Review by fernando-cortez — APPROVED** [review]
~~Just noticing that Medium has a higher LOD Bias & Particle Raycast Budget value than High. This should fall in between Low and High, right?~~

But it's toggling as expected between LODs when switching quality versions on mobile, so that's great.

Edit: Disregard! Will take a look at Quality settings in PR 713.

File: Assets/Models/DungeonPieces.FBX.meta (hunk 1 above)
Comment (jilfranco-unity) [reply Assets/Models/DungeonPieces.FBX.meta:1]: Gotcha, let me rename things

**Review by fernando-cortez — APPROVED** [review]

**Review by SamuelBellomo — APPROVED** [review]

**Pull request merged**